app/Rules/UnauthorizedEmailProviders.php

```diff
@@ -6,3780 +6,34 @@
-            'yep.it',
-            'yermail.net',
-            'yhg.biz',
-            'ynmrealty.com',
-            'yodx.ro',
-            'yogamaven.com',
-            'yoggm.com',
-            'yomail.info',
-            'yoo.ro',
-            'yopmail.com',
-            'yopmail.fr',
-            'yopmail.gq',
-            'yopmail.net',
-            'yopmail.pp.ua',
-            'yordanmail.cf',
-            'you-spam.com',
-            'yougotgoated.com',
-            'youmail.ga',
-            'youmailr.com',
-            'youneedmore.info',
-            'youpymail.com',
-            'your5.ru',
-            'your5.store',
-            'yourdomain.com',
-            'youremail.cf',
-            'yourewronghereswhy.com',
-            'yourlms.biz',
-            'yourspamgoesto.space',
-            'yourtube.ml',
-            'youxiang.dev',
-            'yroid.com',
-            'yspend.com',
-            'ytpayy.com',
-            'yugasandrika.com',
-            'yui.it',
-            'yuoia.com',
-            'yuurok.com',
-            'yxdad.ru',
-            'yxdad.store',
-            'yxzx.net',
-            'yyolf.net',
-            'z-o-e-v-a.ru',
-            'z0d.eu',
-            'z1p.biz',
-            'z86.ru',
-            'zain.site',
-            'zainmax.net',
-            'zaktouni.fr',
-            'zarabotokdoma11.ru',
-            'zasod.com',
-            'zaym-zaym.ru',
-            'zcovz.ru',
-            'zcovz.store',
-            'zcrcd.com',
-            'zdenka.net',
-            'ze.tc',
-            'zebins.com',
-            'zebins.eu',
-            'zehnminuten.de',
-            'zehnminutenmail.de',
-            'zemzar.net',
-            'zepp.dk',
-            'zetmail.com',
-            'zfymail.com',
-            'zhaoqian.ninja',
-            'zhaoyuanedu.cn',
-            'zhcne.com',
-            'zhewei88.com',
-            'zhorachu.com',
-            'zipcatfish.com',
-            'zipo1.gq',
-            'zippymail.info',
-            'zipsendtest.com',
-            'ziragold.com',
-            'zoaxe.com',
-            'zoemail.com',
-            'zoemail.net',
-            'zik.dj',
-            'zipcad.com',
-            'zipcatfish.com',
-            'zipo1.gq',
-            'zippymail.info',
-            'zipsendtest.com',
-            'ziragold.com',
-            'zoaxe.com',
-            'zoemail.com',
-            'zoemail.net',
-            'zoemail.org',
-            'zoetropes.org',
-            'zombie-hive.com',
-            'zomg.info',
-            'zsero.com',
-            'zumpul.com',
-            'zv68.com',
-            'zxcv.com',
-            'zxcvbnm.com',
-            'zymuying.com',
-            'zzi.us',
-            'zzrgg.com',
-            'zzz.com',
-        ]
-    ) {
-        //
+    /**
+     * Create a new rule instance.
+     */
+    public function __construct()
+    {
+        $this->unauthorizedEmailProviders = explode("\n", trim(
```

Comment:
@xiCO2k can you don't do this on the __construct? maybe on the rule itself?

Reply:
sure!